### DISCUSSION TITLE
a lot of usage of f row

### DISCUSSION THREAD

**Discussion by victororlyk** [zed-industries/zed#6503] [Feature Requests]
Honestly I would prefer not to use f row as much as it is used in zed
- rename
- errors navigation
- go to defintion
all three of above are really important for usage, but a lot of users have keyboards like mine https://happyhackingkb.com/ and will be a bit harder to use then competitor editors

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
